components/project-card/project-card.jsx

```diff
@@ -149,7 +149,7 @@ export default React.createClass({
                   : (<div>
                       <Link to={detailViewLink} onClick={this.handleReadMoreClick}>Read more</Link>
                     </div>);
-    let creators = this.props.creators ? `By ${this.props.creators.join(`, `)}` : null;
+    let creators = this.props.creators.length > 0 ? `By ${this.props.creators.join(`, `)}` : null;
```

Comment:
there seem to be a lot of PropTypes missing, so it seems like this component shouldn't be able to rely on there being a `props.creators` at all. Do we want to test for creators, then access length, or do we want to make sure `creators` is part of the known proptypes?